charts/values.yaml

```diff
@@ -0,0 +1,70 @@
+# Default values for EdgelessDB
+
+global:
+  image:
+    pullPolicy: IfNotPresent
+    version: v0.1.1
+    repository: ghcr.io/edgelesssys
+
+  createdBy: Helm
+
+  # Additional annotations to add to all pods
+  podAnnotations: {}
+
+  # Additional labels to add to all pods
+  podLabels: {}
+
+  # For Private docker registries, authentication is needed.
+  #  Registry secrets are applied to the respective service accounts
+  # pullSecret:
+  # pullSecret: my-private-docker-registry-login-secret
+
+edb:
+  replicas: 1
```

Comment:
We allow setting this value, but don't use it in the charts.
I would suggest either removing it for now, since EDB currently only works as a single instance, or consume the value in `templates/edgelessdb.yaml` to set `spec.replicas`